features/standard/SearchContent.feature

```diff
@@ -12,3 +12,17 @@ Feature: Searching for a Content item
     And I open "Dashboard" page in admin SiteAccess
     When I search for a Content named "Searched folder"
     Then I should see in search results an item named "Searched folder"
+
+  @javascript @APIUser:admin @searchudw
```

Comment:
@searchudw is a custom tag for running test purposes or we use it globally?

Reply:
I intend to remove it before merging. I've put a note to do so in the v3 PR checklist.